assets/l10n/en-US.ftl

```diff
@@ -702,6 +704,11 @@ label_chat_call_ongoing = Active call
 label_chat_call_unanswered = Unanswered call
 label_chat_monolog = Your notes
 label_chat_monolog_description = Only you can see this chat. May contain any amount of messages. May contain any type of content.
+label_chat_monolog_features =
```

Comment:
`label_chat_monolog_description` is no longer used in the project. Shouldn't it be just removed? Or better: instead of creating additional l10n key isn't it possible to use the existing one instead?